testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -0,0 +1,86 @@
+    And I select the hostname of "proxy" from "proxies"
+    And I check "manageWithSSH"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "sle_ssh_tunnel_minion"
+
+  Scenario: The contact method is SSH tunnel on this minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    Then I should see a "Push via SSH tunnel" text
+
+  Scenario: Install a package from this SSH tunnel minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    When I follow "Software" in the content area
+    And I follow "Install"
+    And I enter "milkyway-dummy" as the filtered package name
+    And I click on the filter button
+    And I check "milkyway-dummy" in the list
+    And I click on "Install Selected Packages"
+    And I click on "Confirm"
+    Then I should see a "1 package install has been scheduled for" text
+    When I force picking pending events on "sle_ssh_tunnel_minion" if necessary
+    Then I wait until event "Package Install/Upgrade scheduled by admin" is completed
+
+  Scenario: Remove a package from this SSH tunnel minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    And I follow "Software" in the content area
+    And I follow "List / Remove"
+    And I enter "milkyway-dummy" as the filtered package name
+    And I click on the filter button
+    And I check "milkyway-dummy" in the list
+    And I click on "Remove Packages"
+    And I click on "Confirm"
+    Then I should see a "1 package removal has been scheduled" text
+    When I force picking pending events on "sle_ssh_tunnel_minion" if necessary
+    Then I wait until event "Package Removal scheduled by admin" is completed
+
+  Scenario: Run a remote command on this SSH reverse minion
+    Given I am authorized as "testing" with password "testing"
+    When I follow the left menu "Salt > Remote Commands"
+    Then I should see a "Remote Commands" text in the content area
+    When I enter command "echo 'My remote command output'"
+    And I enter the hostname of "sle_ssh_tunnel_minion" as "target"
+    And I click on preview
+    Then I should see a "Target systems (1)" text
+    When I wait until I do not see "pending" text
+    And I click on run
+    And I wait until I see "show response" text
+    And I expand the results for "sle_ssh_tunnel_minion"
+    Then I should see "My remote command output" in the command output for "sle_ssh_tunnel_minion"
+
+  Scenario: Cleanup: delete the SSH push tunnel minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "sle_ssh_tunnel_minion" should not be registered
+
+  Scenario: Cleanup: register a salt minion after SSH push tests
```

Comment:
No need to fix.. but just so you are aware, maybe here we should also say SSH tunnel minion